### ISSUE TITLE
False positive in LongQuoteBlocks

### ISSUE THREAD

**Issue by stepnivlk** [rrrene/credo#531]
### Precheck

* For proposing a new feature, be sure to name your issue "Proposal: ..."
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 0.9.1
* Erlang/Elixir version (`elixir -v`): 1.6.3
* Operating system:  Mac OS

### What were you trying to do?
I have a module which is meant to be `use`d in another module. I have `__using__` with nested `quote` block. Inside of the block, there are methods with comments. Those methods are very lightweight, just a proxy somewhere else. But comments are quite a long and credo considers comments for `LongQuoteBlocks`. So it reports `[F] ↗ Avoid long quote blocks.` on my module just because of those comments.
E.g.:
```elixir
defmodule MyModule
  defmacro __using__(opts) do
    quote bind_quoted: [opts: opts] do
      # Very long comment with e.g. 150 lines
      # ...
      # ...
      def method do
        SomeOtherModule.call()
      end
    end
  end
end
```

### Expected outcome
```
-> mix credo
#> 151 mods/funs, found no issues.
```

### Actual outcome
```
-> mix credo
 Refactoring opportunities
┃
┃ [F] ↗ Avoid long quote blocks.
┃       lib/pushest.ex:99:5 #(Pushest.__using__)
```

**Comment by rrrene** [comment]
Thx for reporting! This is a valid point, we should only count LOC! :+1:

**Comment by stepnivlk** [comment]
@rrrene cool, I can try to work on it if you want.

**Comment by rrrene** [comment]
Thx for reporting this! :+1: It should be be fixed in the latest version of Credo.
